app/models/question.rb

```diff
@@ -1,3 +1,7 @@
 class Question < ApplicationRecord
   belongs_to :user
+  has_many :answers
+
+  validates :user, presence: true
+  validates :content, length: { maximum: 50 }
```

Comment:
Limiting the question makes sense as we want concise questions. Maybe we could make this ~100 to be safe tho?

Reply:
sounds good, let me update